lualatex-math.dtx

```diff
@@ -326,10 +326,10 @@
 %    \begin{macrocode}
```

Comment:
Please also bump the version number and add a changelog entry (using `\changes`).

Reply:
Done.